clang/test/CXX/dcl.dcl/dcl.attr/dcl.attr.nodiscard/p2.cpp

```diff
@@ -17,10 +17,10 @@ E get_e();
 // cxx11-warning@-1 {{use of the 'nodiscard' attribute is a C++17 extension}}
 
 void f() {
-  get_s(); // expected-warning {{ignoring return value of function declared with 'nodiscard' attribute}}
+  get_s(); // expected-warning {{ignoring return value of type 'S' declared with 'nodiscard' attribute}}
```

Comment:
```suggestion
  get_s(); // expected-warning {{ignoring return value type 'S' declared with 'nodiscard' attribute}}
```

Reply:
Umm... the error message does say "ignoring return value of type 'S'", do you want me to remove that of in general?

Reply:
The suggested change was to remove just the `of`.  But reading it again, I'm not agreeing with myself anymore.  Just ignore this request.